src/sknnr_spatial/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.1"
+__version__ = "0.1.0+dev"
```

Comment:
FYI, using a `+` local version specifier was needed to let Pooch know that it should grab data files from the `main` branch on Github. Otherwise, Pooch will assume there's a tag for the current version and try to download from there.

When we make a release, e.g. `0.1.0`, we'll set this accordingly, push that tag to Github, and Pooch will be able to download the data from that tag.